light-client/src/header/eth_header.rs

```diff
@@ -489,6 +431,11 @@ impl TryFrom<RawETHHeader> for ETHHeader {
             None
         };
 
+        #[allow(clippy::absurd_extreme_comparisons)]
+        if PASCAL_TIMESTAMP > 0 && timestamp >= PASCAL_TIMESTAMP && requests_hash.is_none() {
```

Comment:
When `PASCAL_TIMESTAMP` is `0`, it's interpreted as not being Pascal, rather than indicating it's already activated.
It would be helpful to have explanatory comments near where users specify this value, to prevent misunderstandings since this behavior differs from flag management in geth and bsc.

Reply:
I add the comment into README.
https://github.com/datachainlab/parlia-elc/pull/80/commits/e0cc787a33a223b10ae1f0201a86cbc1621a299c